src/main/target/A_TEMPLATE/target.mk.md

```diff
@@ -0,0 +1,12 @@
+F405_TARGETS    += $(TARGET)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5221948)